### PR TITLE
Pagination for quizzes (Candidate)

### PR DESCRIPTION
**Pull request by andreiuta97**
Added pagination to the quiz listing for the Candidates.
If there are less than 5 quizzes, the pagination block does not appear.

TO BE MENTIONED:
I used the pagination-quizzes-listing as "base" branch since I needed some functions already implemented there.

### CODE REVIEW

File: src/views/candidate-quiz-listing.phtml [hunk 1]
```diff
@@ -49,17 +49,29 @@ require_once('partials/head.phtml');
...
-                <a class="page-link" href="#" tabindex="-1" aria-disabled="true">Previous</a>
-            </li>
-            <li class="page-item"><a class="page-link" href="#">1</a></li>
-            <li class="page-item active"><a class="page-link" href="#">2</a></li>
-            <li class="page-item"><a class="page-link" href="#">3</a></li>
-            <li class="page-item">
-                <a class="page-link" href="#">Next</a>
-            </li>
-        </ul>
+        <?php if ($pages['quizzesNumber'] > 5): ?>
```
Comment (noonlit) [inline src/views/candidate-quiz-listing.phtml]: Consider moving the logic that decides the number of pages out of the html. Maybe send a $pagesCount too.

File: src/Controller/AuthenticationController.php [hunk 2]
```diff
@@ -55,7 +55,7 @@ public function redirectAtLogin(Request $request, array $requestAttributes): Res
             $body = Stream::createFromString('');
             $response = new Response($body, '1.1', 301, '');
             /** @var Response $redirect */
-            $redirect = $response->withHeader('Location', 'http://local.quiz.com/candidate/homepage');
+            $redirect = $response->withHeader('Location', 'http://local.quiz.com/candidate/homepage?page=1');
```
Comment (Raresmldvn) [inline src/Controller/AuthenticationController.php:58]: Please make a service/method in abstract controller that deals with redirects.
Also, generate your URLs from requests, hardcoded URLs are good for local testing but when you will switch environments, nothing will work anymore :)

Comment (Raresmldvn) [reply src/Controller/AuthenticationController.php:58]: @andreiuta97 did you move this to another issue or are you going to do it here?

Comment (andreiuta97) [reply src/Controller/AuthenticationController.php:58]: I'm going to make a separate issue for this one if it's ok with you.

**Pull request merged**